make_sh.ts

```diff
@@ -191,7 +191,7 @@ export const makeInstallerScript = ({
       ${platforms
         .map((p) => {
           return `${p.platform}) 
-          download_url="${p.asset.url}" 
+          download_url="${p.asset.browser_download_url}" 
```

Comment:
can you revert this so I can merge the realpath fix first

Reply:
Done